gui/src/themes/stylekit.ts

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright 2023 Avaiga Private Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+export const stylekitTheme = {
+    palette: {
+        // Primary and secondary colors
+        primary: {
+            main: window.taipyConfig?.stylekit?.colorPrimary,
+        },
+        secondary: {
+            main: window.taipyConfig?.stylekit?.colorSecondary,
+        },
+        error: {
+            main: window.taipyConfig?.stylekit?.colorError,
+        },
+        warning: {
+            main: window.taipyConfig?.stylekit?.colorWarning,
+        },
+        success: {
+            main: window.taipyConfig?.stylekit?.colorSuccess,
+        },
+    },
+    typography: {
+        // Custom font
+        fontFamily: window.taipyConfig?.stylekit?.fontFamily,
+        h6: {
+            fontSize: "1rem",
+        },
+    },
+    shape: {
+        borderRadius: window.taipyConfig?.stylekit?.borderRadius,
+    },
+    // Components normalization
+    components: {
+        // Form control
+        MuiFormControl: {
+            styleOverrides: {
+                root: {
+                    // Fill the available width
+                    display: "flex",
+
+                    "&.taipy-selector": {
+                        marginLeft: 0,
+                        marginRight: 0,
+                    },
+
+                    ".taipy-tree &": {
```

Comment:
@JaufreyTpy I don't understand these rules: for taipy-tree and taipy-layout, the components are not embedded in a FormControl ?
(this is valid for taipy-selector though and indeed I will apply it directly to the taipy component)

Reply:
oh sorry, I think I read it wrong.

Reply:
fro taipy-ree and taipy-layout these rules address the included FormControl (ie the TextField inside the taipy-tree Box)